content/post/cars-and-can.md

```diff
@@ -0,0 +1,159 @@
++++
+date = "2017-10-21T13:37:00+01:00"
+description = "Use a cars CAN bus to read live data."
+title = "TED2: Cars and CAN"
+series = ["Ted"]
+slug = "cars-and-can"
+author = "Auke Willem Oosterhoff"
++++
+
+In [previous post]({{< relref "post/obd_ii_elm327_and_python.md">}}) of this
+series I demonstrated how to obtain live data of a car by using an OBD-II
+adapter. I was a bit disappointend about the fact that I could only read a
+handfull of parameters. Together with Wisse Hooghiem I went a step further by
+hooking up in the CAN network of my car. In this post I'll demonstrate how to
+interact with a car's CAN bus in Linux with help of a an Arduino Uno.
+
+## CAN bus
+
+Every car contains electronic chips to control parts of the car. The are called
```

Comment:
As a narrative note, I should start this paragraph with `First, what is CAN?` or something like that. Now, you're starting right away and that does read somewhat strange.